### ISSUE TITLE
Mouse Scrolling isn't working in Plugins Admin

### ISSUE THREAD

**Issue by Shakil-Shahadat** [notepad-plus-plus/notepad-plus-plus#7623]
### Steps to Reproduce the Issue
1. Open Note++
2. Go to Plugins > Plugins Admin
3. Try scrolling using the mouse wheel

My mouse wheel is perfectly okay and works well everywhere else.

Thanks.

**Comment by chcg** [comment]
@Shakil-Shahadat Have you first marked an item of the list before using the mouse wheel? The scrolling is just working if the list has the focus.

**Comment by Shakil-Shahadat** [comment]
I just tried marking an item and tried scrolling by mouse, still didn't scroll.

Btw, I'm on windows 8.1.

**Comment by rddim** [comment]
@Shakil-Shahadat did you use some tool that allows you to scroll inactive windows only with hovering, something like `WizMouse`, `DesktopOK`? Because on my Win 7 I use such tool and I can't scroll in `Plugin Admin` even with selected plugin. When I terminate that tool I can scroll.

**Comment by Shakil-Shahadat** [comment]
@rddim Yes, I use `WizMouse`. Disabling it enabled mouse scrolling when an item is selected.

I think it still needs fixing for two reasons. Firstly, other programs don't need clicking to enable mouse scrolling. Secondly,  `WizMouse` works well with other apps, so I think scrolling should work even with  `WizMouse` enabled.
